test/CSharp7Features.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+
+namespace CSharp7
+{
+  class IsOperator
+  {
+    void IsPatternMatch() {
+      if (3 is int foo) {}
```

Comment:
Please keep everything in the same (long) `AllInOne.cs` file. No need for a specific C# 7 namespace / file.